libs/ui/src/lib/modules-graph/modules-graph.component.scss

```diff
@@ -0,0 +1,47 @@
+.container {
+  position: relative;
+}
+
+.container path {
+  stroke: #fff;
+}
+
+.svg-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.sequence {
+  width: 600px;
```

Comment:
Can we make this more responsive? Why 600px?

Reply:
It becomes somewhat hard to use when too big or too small. Could consider a min-max range maybe?